x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/bulk_actions/translations.ts

```diff
@@ -70,12 +70,12 @@ export function explainBulkEditSuccess(
         x.type === BulkActionEditType.delete_index_patterns
     )
   ) {
-    return `${i18n.RULES_BULK_EDIT_SUCCESS_DESCRIPTION(summary.succeeded)}. ${
-      i18n.RULES_BULK_EDIT_SUCCESS_INDEX_EDIT_DESCRIPTION
+    return `${i18n.RULES_BULK_EDIT_SUCCESS_DESCRIPTION(summary.succeeded, summary.skipped)}${
+      summary.skipped > 0 ? ` ${i18n.RULES_BULK_EDIT_SUCCESS_DATA_VIEW_RULES_SKIPPED_DETAIL}` : ''
```

Comment:
Nit: would be more readable if split into multiple lines

Reply:
Also, why join w/o a space?

Reply:
Refactored this for clarity. The space is included in the `RULES_BULK_EDIT_SUCCESS_DATA_VIEW_RULES_SKIPPED_DETAIL` translation, so it displays only if that translation should be shown (if there are skipped rules)